sda/simulation.go

```diff
@@ -106,35 +106,24 @@ func LoadSimulationConfig(dir, ha string) ([]*SimulationConfig, error) {
 			ha += ":"
 		}
 		for _, e := range sc.Roster.List {
-			for _, a := range e.Addresses {
-				log.Lvl4("Searching for", ha, "in", a)
-				// If we are started in Deterlab- or other
-				// big-server-needs-multiple-hosts, we might
-				// want to initialise all hosts in one instance
-				// of 'cothority' so as to minimize memory
-				// footprint
-				if strings.Contains(a, ha) {
-					log.Lvl3("Found host", a, "to match", ha)
-					host := NewHost(e, scf.PrivateKeys[a])
-					scNew := *sc
-					scNew.Host = host
-					scNew.Overlay = host.overlay
-					ret = append(ret, &scNew)
-				}
-
-			}
+			host := NewHost(e, scf.PrivateKeys[e.Address])
```

Comment:
Is this still correct? IIRC this was to set up all `Conode`s that have to be run on the same machine. But here you take _all_ `Conode`s defined by the system.